File: 026.RemoveDuplicatesFromSortedArray/wyrj/RemoveDuplicatesFromSortedArray.c
/* 12ms */
int removeDuplicates(int* nums, int numsSize) {
    if (0 == numsSize) {
        return 0;
    }
    int index = 0;
    for (int i = 1; i < numsSize; i++) {
        if (nums[index] != nums[i]) {
            index ++;
            nums[index] = nums[i];
        }
    }
    return index + 1;
}

File: 026.RemoveDuplicatesFromSortedArray/wyrj/RemoveDuplicatesFromSortedArray.js
/* 180ms */
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    var index = 0, len = nums.length;
    for (var i = 1; i < len; i++) {
        if (nums[i] !== nums[index]) {
            index++;
            nums[index] = nums[i];
        }
    }
    return index + 1;
};